src/python/pants/build_graph/address.py

```diff
@@ -234,21 +233,7 @@ def __init__(
         self._target_name: Optional[str]
         if target_name and target_name != os.path.basename(self.spec_path):
             banned_chars = BANNED_CHARS_IN_TARGET_NAME & set(target_name)
-            deprecated_banned_chars = banned_chars & set(r"/\:")
```

Comment:
I think this is the correct change, but please double-check.

Reply:
👍